Assets/Scripts/Shop/Item.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Item
+{
+    public enum ItemType
```

Comment:
What are these? Refer to the shop items listed here: https://docs.google.com/document/d/1PvrM6-0IUvAIAV-rXJwFxBCR__dQ7gjV3qBFxtDPfjs/edit

Reply:
these were placeholders